### PR TITLE
Override getExternalHeadingVelocity

### LINKED ISSUE #124: Missing getHeadingVelocity method
When running the ManualFeedforwardTuner, the program threw an exception. The error message was "poseVelocity() must not be null. Ensure that the getWheelVelocities() method has been overridden in your localizer." It refers to ManualFeedforwardTuner.java:119. Using the debugger, we traced the issue to the heading velocities being null when the pose velocity is set. It seems like there should be a getHeadingVelocity() method in the localizer method (we use StandardTrackingWheelLocalizer) to correct this, but there isn't. getWheelVelocities() has been overridden, so that is not the issue. Is there a reference implementation for getHeadingVeolocity() somewhere or a standard way to implement it? We are using two wheel odometry with the built-in Rev Hub IMU.

### PR DESCRIPTION
**Pull request by NoahBres**
Fixes #124

### CODE REVIEW

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/SampleMecanumDrive.java [hunk 1]
```diff
@@ -394,4 +394,28 @@ public void setMotorPowers(double v, double v1, double v2, double v3) {
...
+        //                           |
+        //     (Motor Port Side)     |   / X axis
+        //                       ____|__/____
+        //          Y axis     / *   | /    /|   (IO Side)
+        //          _________ /______|/    //      I2C
+        //                   /___________ //     Digital
+        //                  |____________|/      Analog
+        //
+        //                 (Servo Port Side)
+        //
```
Comment (rbrott) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/SampleMecanumDrive.java:411]: Impressive sketch!

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/SampleMecanumDrive.java [hunk 2]
```diff
@@ -394,4 +394,28 @@ public void setMotorPowers(double v, double v1, double v2, double v3) {
...
+        // In this diagram, the side normal to the x axis facing away from you is the side with the
+        // USB port(s)
```
Comment (rbrott) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/SampleMecanumDrive.java:413]: Is it clearer to say "the positive x axis points toward the USB port(s)"?

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/SampleMecanumDrive.java (hunk 1 above)
Comment (NoahBres) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/SampleMecanumDrive.java:411]: Base ascii axis art shamelessly stolen from some Adafruit forum post that I can no longer source.

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/SampleMecanumDrive.java (hunk 2 above)
Comment (NoahBres) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/SampleMecanumDrive.java:413]: Would that your statement verbatim be more clear?

Comment (rbrott) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/SampleMecanumDrive.java:413]: Yes, that phrasing suffices verbatim.

**Pull request merged**